install-appliance.md

```diff
@@ -116,9 +156,13 @@ The appliance requires a few additional tools.
 
 After the installation, access `https://your-machine-address`. This web page provides status information for the following initialisation process.
 
+> **NOTE:** By default, the Anbox Cloud Appliance uses self-signed certificates, which might cause a security warning in your browser. Use the mechanism provided by your browser to proceed to the web page.
+
 ![appliance-welcome|690x343, 100%](upload://yIGZThPljsjPyRAVQVFkZOiVVNF.png)
 
-Invoke the initialisation process of the Anbox Cloud Appliance via SSH on the machine that hosts the appliance:
+Log on to the machine that hosts the appliance. If you installed on an AWS instance, see [Connect to your Linux instance using SSH](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html) for instructions.
```

Comment:
This link might confuse users as the Ubuntu image doesn't have an `ec2-user` user, just the `ubuntu` user. We should either tell the user to simply run `ssh ubuntu@<machine-ip>` or point to that article but specify that the user should be `ubuntu` instead of `ec2-user`

Reply:
Just "ssh ubuntu@<machine-ip>" doesn't work, because you need to specify the key file. Not sure if that can be prevented by naming the file differently? 
I definitely needed the help article to figure out what to do. But sure, I can point out the user name.

Reply:
On ubuntu renaming the key to `id_rsa` does the trick, or you can use `ssh -i $HOME/.ssh/aws-ssh-key.pem ubuntu@foobar.com`
Let's leave the article then, with a hint to the proper username

Reply:
Added information about the user name and key path. Seems the only way to get it to work without specifying the path is to rename the key file, and that is not good usability since everybody will already have a file with that name in the .ssh folder ... :(